governance/program/src/state.rs

```diff
@@ -0,0 +1,225 @@
+//! Program accounts
+
+use solana_program::pubkey::Pubkey;
+
+/// Max length of a governance name
+pub const MAX_GOVERNANCE_NAME_LENGTH: usize = 32;
+
+/// Max length of a proposal description link
+pub const MAX_PROPOSAL_DESCRIPTION_LINK_LENGTH: usize = 200;
+
+/// Max length of a proposal name
+pub const MAX_PROPOSAL_NAME_LENGTH: usize = 32;
+
+/// Max number of transactions allowed for a proposal
+pub const MAX_TRANSACTIONS: usize = 5;
```

Comment:
I think you mean Instructions instead of Transactions.   A program (this program) can issue instructions but not transactions

Reply:
Changed to instruction.